target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -128,6 +128,7 @@ get_status_led() {
 	wl-330n|\
 	wl-330n3g|\
 	wli-tx4-ag300n|\
+	c20|\
```

Comment:
please keep alphabetical order. You might need to move whole block to keep the overall order.